src/parachain/loans.ts

```diff
@@ -375,16 +319,139 @@ export class DefaultLoansAPI implements LoansAPI {
             })
         );
 
-        return <LendPosition[]>allMarketsPositions.filter(position => position !== null);
+        return <LendPosition[]>allMarketsPositions.filter((position) => position !== null);
     }
 
     getBorrowPositionsOfAccount(accountId: AccountId): Promise<Array<BorrowPosition>> {
         return Promise.resolve(MOCKDATA_BORROW_POSITIONS);
     }
 
+    async _getLoanAssetLendApy(underlyingCurrencyId: InterbtcPrimitivesCurrencyId): Promise<Big> {
```

Comment:
Why not just
```suggestion
    async _getLendApy(underlyingCurrencyId: InterbtcPrimitivesCurrencyId): Promise<Big> {
```

The `LoanAsset` part can be inferred from the parameter name / type.